code/modules/mob/living/carbon/xenomorph/castes/Defender.dm

```diff
@@ -36,6 +36,7 @@
 		/datum/action/xeno_action/onclick/xeno_resting,
 		/datum/action/xeno_action/onclick/regurgitate,
 		/datum/action/xeno_action/watch_xeno,
+		/datum/action/xeno_action/activable/tail_stab,
```

Comment:
defender tails have their own tail ability and are clubbed not spiked

Reply:
removed it